crypto/rsa/rsa_locl.h

```diff
@@ -28,7 +28,7 @@ DEFINE_STACK_OF(RSA_PRIME_INFO)
 struct rsa_st {
     /*
      * The first parameter is used to pickup errors where this is passed
-     * instead of aEVP_PKEY, it is set to 0
+     * instead of a EVP_PKEY, it is set to 0
```

Comment:
personally I would say "an EVP_PKEY"

Reply:
Yeah, that does sound better. I've updated this now. Thanks